clearwater_config_manager/sas_json_plugin.py

```diff
@@ -50,5 +75,53 @@ def on_config_changed(self, value, alarm):
             run_command(["/usr/share/clearwater/bin/reload_sas_json"])
             alarm.update_file(_file)
 
+    def resolve_domain_address(self, addr, dns_server=None): # pragma: no cover
```

Comment:
I think both this and resolve_domain_address_in_namespace shouldn't be part of the class - they should probably just be standalone functions in this file.